lms/static/sass/base/_grid-settings.scss

```diff
@@ -4,10 +4,13 @@
 $max-width: 1200px;
 /* Override the default global box-sizing */
 $border-box-sizing: false;
+/* Grid width variables */
+$large-min-width: 769px;
 
 
 /* Breakpoints */
-$mobile: new-breakpoint(max-width 320px 4);
-$tablet: new-breakpoint(min-width 321px max-width 768px, 8);
-$desktop: new-breakpoint(min-width 769px 12);
-$xl-desktop:  new-breakpoint(min-width 980px 12);
+$tiny: new-breakpoint(max-width 320px 4);
```

Comment:
Thanks for converting these. Since they are not included in a Sass map which gives context, can you prefix the variables with a ``$bp-*`` to help give context to developers using them? My fear is that someone would use them inappropriately (for font-sizes or other uses).